content/_index.en.md

```diff
@@ -8,10 +8,10 @@ hero:
       author: OECD Observatory of Public Service Innovation
       title: Rules as Code enables “better policy outcomes and enhanced service delivery”
       link: https://oecd-opsi.org/publications/cracking-the-code/
-    - image: dpg.png
-      author: Digital Public Goods Alliance
-      title: Digital Public Good Nominee
-      link:
+    - image: wgs.png
+      author: World Governement Summit
+      title: Edge of Government Innovation Award 2023
+      link: https://edge.worldgovernmentsummit.org
```

Comment:
When looking at that website, it will certainly be obsolete next year. I didn't find a way to link specifically to this year's entries. I thus suggest we use the [video presentation](https://www.youtube.com/watch?v=gx6SDXe-E74) or [this newspaper article](https://www.khaleejtimes.com/uae/sheikh-saif-honours-french-governments-openfisca-project).

Reply:
I used the link to the article, I hope there will be a publication on the official website.